### PR TITLE
Add vtex-css-utils

### PR DESCRIPTION
**Pull request by dfosco**
Adds css-utils to master branch

### CODE REVIEW

File: src/utils/base/theme.less [hunk 1]
```diff
@@ -0,0 +1,35 @@
+@import "theme-variables";
+
+.theme-font {
+  font-family: 'Lato';
+}
+
+h1, .h1 {
```
Comment (brenoc) [inline src/utils/base/theme.less]: Why not only use the `.h1`?

Comment (dfosco) [inline src/utils/base/theme.less]: @brenoc My idea was to make general styles more easily available — for example, an app developer could user the class `.h1` to pull the theme's `<h1>` style... but I guess that might just push developers into doing less semantic code.

Should we scrap it? I'd be ok with it.

Comment (brenoc) [inline src/utils/base/theme.less]: Actually I prefer just using `.h1` and leaving h1 tags styling alone. Less handy, but more safe from external/global styling.

Comment (dfosco) [inline src/utils/base/theme.less]: That's a good idea, safer for everybody involved — we can't enforce it, but by setting up the theme this way and referencing in the docs we have a good change people will follow it.

Comment (dfosco) [inline src/utils/base/theme.less]: Just remember you'll break the theme if you remove the `h1`, `h2`, etc — I'd do this change in another PR.

Comment (brenoc) [inline src/utils/base/theme.less]: I think it could be in this one. Since we don't want pul requests to break anything.

**Comment by tamorim** [comment]
@dfosco Just to remember! :)

**Comment by brenoc** [comment]
@dfosco!

**Comment by dfosco** [comment]
@brenoc @tamorim Closing this PR — [#108](https://github.com/vtex-apps/theme/pull/108) contains the changes listed here and more

**Comment by tamorim** [comment]
@dfosco can we delete this branch?

**Comment by dfosco** [comment]
@tamorim yep!
